docs/guides/how-to/general.md

```diff
@@ -0,0 +1,48 @@
+Here are several pointers to other places in the docs, for general or frequent questions.
+
+
+### Set a Service Name
+To see your service names in the logfire UI, you just need to set the `service_name` parameter in
+[`logfire.configure()`][logfire.configure(service_name)].
```

Comment:
This is easy to search for, it's one of the first parameters of `logfire.configure`, and the name is self-explanatory. Is this section helping? Did people ask about this? I think what's missing is some motivation (e.g. a screenshot showing the column of bubbles/tags for different services) and an explanation of when you'd use it (different kinds of processes).

Reply:
I think this is classic "we're too used to our own product" issues. When I first joined this wasn't obvious to me initially - none of our examples show it (which is what most users look at)

Reply:
When I say the name is self-explanatory, I mean that it's obvious that the `service_name` parameter sets the service name. Nothing in this section tells the user more than that, i.e. what the service name means and how to set it. If when you first joined you were wondering "how do I get those bubbles I've been seeing", this wouldn't tell you that. A new user might not even know those bubbles are possible. I'd like to solve that. Maybe adding more content here along those lines would help, maybe not.